llvm/test/Analysis/ScalarEvolution/pr87798.ll

```diff
@@ -0,0 +1,59 @@
+; RUN: opt -disable-output -passes='print<scalar-evolution>' -verify-scev < %s 2>&1 | FileCheck %s
```

Comment:
Could you please use update_analyze_test_checks.py to generate the output?

Reply:
it doesn't generate the SCEV lines unfortunately. I had tried it initially and then ended up generating these CHECK lines manually. 

The only lines it added were at end of test:
```
+ ;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
+ ; CHECK: {{.*}}
```

Reply:
ah!! I just noticed update_analyze_test_checks in your comment! I was trying update_test_checks. Let me try that